### DISCUSSION TITLE
Anyone done a OrcaSlicer .json config bundle yet?

### DISCUSSION THREAD

**Discussion by 64bitcube** [bassamanator/Sovol-SV06-firmware#104] [General]
I'm planning on doing my klipper switch-over when time presents itself in the next couple weeks, and I had been eyeing up OrcaSlicer in lieu of superslicer not actively updating.  Unfortunately it doesn't import .ini's.  I've found theophile's script for converting Prusa/SS ini to json, but in the interest of saving myself some grief I wonder if anyone's already done a conversion of Bassamanator's config or a scratch profile that they can confirm works?  

Worst case, I'll report back after giving it a go.  I'll just have to take my time with it since I'll be fresh to both the slicer and perl, and there will likely be a few formatting hiccups.

**Comment by bassamanator** [comment]
You'll find orcaslicer profiles by `joe es*` on printables.

**Reply by 64bitcube** [reply]
His profile is for the SV06-Plus from what I can see, though I do like the buildplate model more than the one included stock with Orca. I keep forgetting there's stuff on printables besides models.

Blakadder's SV06 website also has a klipper profile for a .6mm nozzle.  I was planning on keeping at least one machine on a .4 nozzle, but it could be a very good place to start.

**Reply by bassamanator** [reply]
Why not just adjust the build volume to your needs? That's what i did when i explored OS. In any case, happy printing.

**Reply by 64bitcube** [reply]
Build volume should be easy, but I assume there will be additional changes given the SV06 plus has a larger heat block & sovol's volcano style nozzle, so the flow rate / speeds will need to be re-tuned.  I know that klipper/the printer will handle the PID so no need to fuss there, but I assume the slicer will still be trying to embed some speed info into the gcode.  Again, I'm coming from Marlin so it could be I'm missing something glaring in the logic of what (machine, server, gcode, etc) is handling each part of the settings.

Regardless, thanks for the help!

**Reply by brogrene** [reply]
I think this is the SV06 Plus profile you are referring:
 (@joescalon_263091(https://www.printables.com/@joescalon_263091) [SV06 Plus Profiles and Build Plate](https://www.printables.com/model/508400-sv06-plus-profiles-and-build-plate) )

I'm new at all this but seems to work fine and much, much faster. Besides build volume the PRINT_START parameters in the Machine start G-code within Orca wouldn't work. I needed to use:  PRINT_START BED=[first_layer_bed_temperature] HOTEND=[first_layer_temperature[initial_extruder]]

**Reply by dasb00ter** [reply]
@brogrene was this a complete solution I'm struggling with a solution for this I'm using joescalon profile and contacted him and he actually calls 3dprintdemon(GitHub) macros

**Reply by brogrene** [reply]
It's been awhile and I've been trying out other things but I think all I did was change the start code in Orca. But here's what I currently have:
**Machine start G-Code**
G90 ; use absolute coordinates
M83 ; extruder relative mode

M104 S0 
M140 S0
PRINT_START BED=[first_layer_bed_temperature] HOTEND=[first_layer_temperature[initial_extruder]]
PURGE_LINE

**Machine end G-Code**
PRINT_END
